addons/web/static/src/legacy/js/fields/relational_fields.js

```diff
@@ -3200,6 +3200,42 @@ var FieldStatus = AbstractField.extend({
             });
         }
     },
+
+    on_attach_callback() {
+        const self = this;
+        if(self.isClickable==true){
+            if (self.viewType === "form") {
+                let provide = () => {
+                    return self.status_information.map((value) => ({
+                        name: value['display_name'],
+                        action: () => {
+                            this._setValue(value['id']);
+                        }
+                    }))
+                }
+                let getCommandDefinition = (env) => ({
+                    name: env._t("Move to stage..."),
```

Comment:
Not sure the term "Stage" is generic enough. In the codebase, there are a lot of `<field name="state" widget="statusbar"/>`.